code/drasil-gool/lib/Drasil/Shared/InterfaceCommon.hs

```diff
@@ -108,6 +110,7 @@ class (TypeSym r) => VariableSym r where
   extVar    :: Library -> Label -> VSType r -> SVariable r
   arrayElem :: Integer -> SVariable r -> SVariable r
 
+-- Similar here - shouldn't we be handling this on a per-language basis?
```

Comment:
classes exactly let you do things per-language. Now, if you mean that each language might want to vary this signature, that's different!